packml_devel/CMakeLists.txt

```diff
@@ -0,0 +1,187 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(packml_devel)
```

Comment:
In the end, I don't think we will stick with the package name `packml_devel`.  Because the package name can be painful to change later, I recommend starting off with the desired name.  `packml_smach` is probably a good name.